ecosystem/sep-0031.md

```diff
@@ -86,7 +86,7 @@ All endpoints respond with content type:
 1. The Sending Anchor makes [SEP-12 PUT /customer](sep-0012.md#customer-put) requests containing the collected information for each client.
 1. On successful registration (`202` HTTP status), the Sending Anchor makes a `POST /transactions` request to create a transaction record with the Receiving Anchor. This request contains the `id`s returned by the `PUT /customer` requests, the `transaction.fields` values collected from the Sending Client, as well as the transaction amount and asset information. The response will include a `id` that will be used to check the status of the transaction.
 1. The Sending Anchor makes `GET /transactions/:id` requests until the transaction's `status` is `pending_sender`. 
-1. The Sending Anchor submits the payment transaction to Stellar.
+1. The Sending Anchor submits the payment transaction to Stellar using the memo and memo_type specified by the Receiving Anchor `GET /transactions/:id` response fields `stellar_memo` and `stellar_memo_type`.
```

Comment:
Since this clarification leaves out the `stellar_account_id` response parameter, I think it's better to say the following:

```suggestion
1. The Sending Anchor submits the payment transaction to Stellar using the information provided in the `POST /transactions` response.
```

Reply:
https://github.com/stellar/stellar-protocol/pull/962/commits/37e9d9030c066f8aa6826509a17a68f0eda64fd6